Components/image/image.css

```diff
@@ -0,0 +1,15 @@
+.res-img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.sq-img {
+  width: 11rem;
+  height: 11rem;
+  object-fit: cover;
```

Comment:
Instead of declaring object-fit:cover separately in all classes you can add it commonly.

Reply:
I can but for one css property making a class is not worth it , as Vikram also said in the class